awx_collection/plugins/modules/tower_workflow_job_template_schema.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+
+# (c) 2020, John Westcott IV <john.westcott.iv@redhat.com>, Sean Sullivan <ssulliva@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: tower_workflow_job_template_node
+author: "John Westcott IV (@john-westcott-iv), Sean Sullivan (@Wilk42)"
+version_added: "2.9"
+short_description: create, update, or destroy Ansible Tower workflow job template graphs.
+description:
+    - Create, update, or destroy Ansible Tower workflow job template graphs.
+    - Use this to build a graph for a workflow, which dictates what the workflow runs.
+    - Replaces the deprecated tower_workflow_template module schema command.
+    - This module takes a josn list of inputs to first create nodes, then link them, and does not worry about ordering.
+      For failsafe referencing of a node, specify identifier, WFJT, and organization.
+      With those specified, you can choose to modify or not modify any other parameter.
+options:
+    workflow_job_template:
+      description:
+        - The workflow job template the node exists in.
+        - Used for looking up the node, cannot be modified after creation.
+      required: True
+      type: str
+      aliases:
+        - workflow
+    organization:
+      description:
+        - The organization of the workflow job template the node exists in.
+        - Used for looking up the workflow, not a direct model field.
+      type: str
+    schema:
+      description:
+        - A json list of nodes and their coresponding options.
+      type: str
```

Comment:
Why would you take this as a JSON str? If it's ultimately a list, then use the "list" type, and since it's a list of nodes, I'm guessing that the elements (specified by the `elements:` key here) are "dict".